policyengine_us/parameters/gov/states/dc/tax/income/credits/ctc/income_threshold.yaml

```diff
@@ -0,0 +1,56 @@
+description: DC limits its Child Tax Credit to filers with taxable income below the following amount, based on filing status. 
+metadata:
+  label: DC Child Tax Credit taxable income threshold
+  unit: currency_USD
+  period: year
+  breakdown:
+    - filing_status
+  reference:
+    - title: Code of the District of Columbia § 47-1806.17(e)(5)
+      href: https://code.dccouncil.gov/us/dc/council/code/sections/47-1806.17#(e)(5)
+    
+SINGLE:
+  values:
+    2025-01-01: 160_000
+  metadata:
+    uprating:
```

Comment:
Would propagate_metadata_to_children work here to avoid duplicating?

Reply:
I don't think so, I believe we have tried it in the past. Do we have documentation on how `propagate_metadata_to_children` works for other devs?